thunderbird/src/main/java/zeta/android/thunderbird/api/devapi/response/pdp/PdpSkuAvailabilityDetailMapResponseDeserializer.java

```diff
@@ -0,0 +1,37 @@
+package zeta.android.thunderbird.api.devapi.response.pdp;
+
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+
+import java.lang.reflect.Type;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+
+
+public class PdpSkuAvailabilityDetailMapResponseDeserializer implements JsonDeserializer<PdpSkuAvailabilityDetailMapResponse> {
+
+    @Override
+    public PdpSkuAvailabilityDetailMapResponse deserialize(JsonElement json, Type typeOfT, JsonDeserializationContext context) throws JsonParseException {
+
+        Gson gson = new GsonBuilder().create();
```

Comment:
We should use injection pattern. To get Gson here. let me see how I can clean it up.